### ISSUE TITLE
[Bug] [Arch] Window from sketch produces incorrect hole

### ISSUE THREAD

**Issue by FreeCAD-Bug-Importer** [FreeCAD/FreeCAD#6165]
Issue imported from https://tracker.freecad.org/view.php?id=4575

* **Reporter:** AndrewA
* **Date submitted:** 2/25/2021
* **FreeCAD version:** 0.2
* **Category:** Bug
* **Status:** confirmed
* **Tags:** 

# Original report text

When putting a window in a roof, the hole often works, but at times it produces the wrong hole.

https://forum.freecadweb.org/viewtopic.php?f=23&t=56018&p=481865#p481865

# Additional information

This is a minor bug as there is a work around, see the forum post above

# Steps to reproduce

To create the window I had a standard window in first, then changed the sketch to a custom one.
Often this works fine, as the example attached did until I made it higher,then all changes to the image seem to not do anything. I cannot reliably reproduce the behaviour, although it has happened before.

# FreeCAD Info

```
OS: Debian GNU/Linux bullseye/sid (XFCE/xfce)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.24212 (Git) AppImage
Build type: Release
Branch: master
Hash: 3d227cf7179c00e57f2733f947b4b1b0368d219a
Python version: 3.8.6
Qt version: 5.12.5
Coin version: 4.0.0
OCC version: 7.4.0
Locale: English/United Kingdom (en_GB)
```

# Other bug information

* **Priority:** low
* **Severity:** minor
* **Category:** Bug
* **Updated:** 2/26/2021

# Discussion from Mantis ticket

### Comment by Kunda1 2021-02-26 12:12
Assigning to yorik

**Comment by luzpaz** [comment]
Asked OP on the forum thread to report if this is reproducible on v0.20dev

**Comment by luzpaz** [comment]
CC @paullee0

**Comment by Roy-043** [comment]
Fixed with #6617.

**Comment by paullee0** [comment]
Thanks @Roy-043
